views/partials/header.hbs

```diff
@@ -1,9 +1,15 @@
 <div class="header">
-  <a href="/"><h1 class="text-center naz-events">Nazareth Events</h1></a>
-  <a href="/add-event" class="glyphicon glyphicon-plus"></a>
+  <div class="div-plusContainer">
+    <a href="/add-event"><img src="./images/plus.png" class="plusIcon" alt="add event"> </a>
+  </div>
+  <a href="/"><span class="naz-events">Nazareth Events</span></a>
   {{#if calendarButton}}
-    <div id="calendar-button" class="div-calendarBtn">
-      <span class="glyphicon glyphicon-calendar calendar"></span>
-    </div>
+      <div class="div-calContainer">
+        <img src="./images/calendar.png" id="calendarIcon" class="calendar"/>
```

Comment:
It would be better to use html icons rather than images - check out font awesome [here](https://www.w3schools.com/icons/icons_reference.asp).

Find the icon and then click 'try it' - it shows you how to add them. Basically include a stylesheet in `main.hbs` and then add class names to get the specific icons.